openpype/hosts/nuke/plugins/publish/validate_asset_context.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""Validate if instance asset is the same as context asset."""
+from __future__ import absolute_import
+
+import pyblish.api
+
+import openpype.hosts.nuke.api.lib as nlib
+
+from openpype.pipeline.publish import (
+    RepairAction,
+    ValidateContentsOrder,
+    PublishXmlValidationError,
+    OptionalPyblishPluginMixin,
+    get_errored_instances_from_context
+)
+
+
+class SelectInvalidNodesAction(pyblish.api.Action):
+    """Select invalid nodes."""
+
+    label = "Select Failed Node"
+    icon = "briefcase"
+    on = "failed"
+
+    def process(self, context, plugin):
+        if not hasattr(plugin, "select"):
+            raise RuntimeError("Plug-in does not have repair method.")
+
+        # Get the failed instances
+        self.log.debug("Finding failed plug-ins..")
+        failed_instance = get_errored_instances_from_context(context, plugin)
+        if failed_instance:
+            self.log.debug("Attempting selection ...")
+            plugin.select(failed_instance.pop())
```

Comment:
Please use `openpype.hosts.nuke.api.action.SelectInvalidAction` instead of reinventing the wheel.

By the way, there's a bug [here](https://github.com/ynput/OpenPype/blob/2b23b42da65b6943a6b46253816422f71c7ac8c6/openpype/hosts/nuke/api/actions.py#L39) - it should be:
```python
                    invalid.extend(invalid_nodes)
```
So that the validation plugin can actually return multiple nodes to be selected.
Fun fact - the action exists, but is completely unused within Nuke. 🤯  So it should even be safe to fix this.